test/SILGen/wildcard_dispatch_on_catch.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: $target-swift-frontend -emit-silgen -verify %s
```

Comment:
-verify is only there to check '// expected-error' that you see in Sema diagnostics tests. You don't need it here.